baseplate/frameworks/queue_consumer/kafka.py

```diff
@@ -99,14 +129,26 @@ def __init__(
         handler_fn: Handler,
         message_unpack_fn: KafkaMessageDeserializer,
         on_success_fn: Optional[Handler] = None,
+        bootstrap_servers: str = "",
     ):
         self.baseplate = baseplate
         self.name = name
         self.handler_fn = handler_fn
         self.message_unpack_fn = message_unpack_fn
         self.on_success_fn = on_success_fn
+        # as far as I can tell, there is no way to extract the bootstrap servers from the
+        # consumer or the message itself. So wee need to have it passed from the parent
+        self.bootstrap_servers = bootstrap_servers
 
     def handle(self, message: confluent_kafka.Message) -> None:
+        logger.error("handling stuff")
+        prom_labels = KafkaConsumerPrometheusLabels(
+            kafka_address=self.bootstrap_servers,
+            kafka_topic=message.topic() if message.topic() is not None else "",
+        )
+        prom_success = "true"
```

Comment:
This implies that success is labeled differently if it were not prometheus. Assuming that's not accurate, can we drop the prefix here?

Reply:
That only implies that this is how we track the prometheus success. Given that this value is only used by prometheus, I feel like this is accurate.